tests/base.py

```diff
@@ -0,0 +1,205 @@
+import tap_tester.connections as connections
+import tap_tester.menagerie as menagerie
+import tap_tester.runner as runner
+import os
+import unittest
+from datetime import datetime as dt
+import time
+
+class AsanaBase(unittest.TestCase):
+    START_DATE = ""
+    DATETIME_FMT = {
+        "%Y-%m-%dT%H:%M:%SZ",
+        "%Y-%m-%d %H:%M:%S",
+        "%Y-%m-%dT%H:%M:%S.%fZ"
+    }
+    PRIMARY_KEYS = "table-key-properties"
+    REPLICATION_METHOD = "forced-replication-method"
+    REPLICATION_KEYS = "valid-replication-keys"
+    FULL_TABLE = "FULL_TABLE"
+    INCREMENTAL = "INCREMENTAL"
+    OBEYS_START_DATE = "obey-start-date"
+
+    first_start_date = '2019-01-01T00:00:00Z'
+    second_start_date = '2020-08-15T00:00:00Z'
+
+    def name(self):
```

Comment:
The base file should not have a name attribute. We want each test to have a unique name, and by adding it to base.py we create the potential for new tests to be added without a name attr and still pass locally.

Reply:
Removed the function.